### PR TITLE
Update mini cart component

### PR DESCRIPTION
**Pull request by vitoria**
#### What is the purpose of this pull request?
Update the mini cart component

#### What problem is this solving?
Add mini cart item link to the product page and update the tests.

#### How should this be manually tested?
```npm t```

#### Screenshots or example usage
```https://minicartnew--storecomponents.myvtex.com/_v/minicart```

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/MiniCart.js [hunk 1]
```diff
@@ -1,97 +1,61 @@
...
+        <Button
+          icon
+          onClick={this.handleClickButton}
+          onMouseEnter={this.handleMouseEnterButton}
+          onMouseLeave={this.handleMouseLeaveButton}>
+          <CartIcon fillColor={miniCartIconColor} />
+        </Button>
+        {(isMouseOnMiniCart || isMouseOnButton) &&
+          <div
+            className="absolute right-0 vtex-minicart__box bg-white z-max"
```
Comment (claudivanfilho) [inline react/MiniCart.js]: opa

Comment (claudivanfilho) [reply react/MiniCart.js]: e esse z-max é perigoso, tem como colocar 3 ou 4?

File: react/MiniCartContent.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  )
+
+  renderMiniCartWithItems = (orderForm, label) => (
+    <div className="flex flex-column" >
+      <div className="vtex-minicart__arrow-up self-end mr3 pr1"></div>
+      <div className="shadow-3">
+        <div className="vtex-minicart__content pa4 overflow-auto">
+          {orderForm.items.map(item => (
+            <div className="flex flex-row" key={item.id}>
+              <MiniCartItem
```
Comment (claudivanfilho) [inline react/MiniCartContent.js]: Deixa em 1 linha só

File: react/MiniCartItem.js [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+import { Link } from 'render'
+
 import './global.css'
 
 /**
  * Minicart item component
  */
 export default class MiniCartItem extends Component {
   static propTypes = {
+    /* Item s id */
```
Comment (claudivanfilho) [inline react/MiniCartItem.js]: Item s

File: react/MiniCartItem.js [hunk 4]
```diff
@@ -18,31 +22,51 @@ export default class MiniCartItem extends Component {
...
+            <div className="vtex-minicart__item-name mt3 tl">
+              <span className="b">{name}</span>
             </div>
-            <div className="absolute right-0 bottom-0 mb3">
-              <Price
-                sellingPrice={sellingPrice}
-                listPrice={listPrice}
-                showLabels={false}
-                showListPrice={false} />
+            <div>
```
Comment (claudivanfilho) [inline react/MiniCartItem.js]: acho que pode tirar essa div

**Review by claudivanfilho — CHANGES_REQUESTED** [review]

File: react/MiniCartItem.js (hunk 3 above)
Comment (vitoria) [reply react/MiniCartItem.js]: Done!

File: react/MiniCartItem.js (hunk 4 above)
Comment (vitoria) [reply react/MiniCartItem.js]: Done!

File: react/MiniCartContent.js (hunk 2 above)
Comment (vitoria) [reply react/MiniCartContent.js]: Done!

**Review by claudivanfilho — APPROVED** [review]

File: react/package.json [hunk 5]
```diff
@@ -3,24 +3,53 @@
   "version": "0.0.1",
```
Comment (brunojdo) [inline react/package.json]: remove

**Review by brunojdo — CHANGES_REQUESTED** [review]

**Review by brunojdo — COMMENTED** [review]
Add `CHANGELOG.md`

Add travis badge on `README.md`

**Review by brunojdo — CHANGES_REQUESTED** [review]
Add `CHANGELOG.md`

Add travis badge on `README.md`

File: react/package.json (hunk 5 above)
Comment (vitoria) [reply react/package.json]: Done!

File: README.md [hunk 6]
```diff
@@ -3,6 +3,10 @@ VTEX Minicart Component
 
 This is an extension point component.
 
+### Travis CI 
+
+[![Build Status](https://travis-ci.org/vtex-apps/menu.svg?branch=master)](https://travis-ci.org/vtex-apps/menu)
```
Comment (brunojdo) [inline README.md]: minicart

**Review by brunojdo — CHANGES_REQUESTED** [review]

File: react/MiniCartContent.js [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+    labelMiniCartEmpty: PropTypes.string,
+    /* Label to appear in the finish shopping button */
+    labelButton: PropTypes.string,
+    /* Internationalization */
+    intl: intlShape.isRequired,
+  }
+
+  handleClickButton = () => location.assign('/checkout/#/cart')
+
+  renderWithoutItems = (label) => (
```
Comment (brunojdo) [inline react/MiniCartContent.js]: without ()

**Review by brunojdo — CHANGES_REQUESTED** [review]

**Review by brunojdo — APPROVED** [review]

**Pull request merged**